monai/utils/aliases.py

```diff
@@ -4,8 +4,8 @@
 import importlib
 
 
-aliasLock = threading.RLock()
-globalAliases = {}
+AliasLock = threading.RLock()
```

Comment:
This is not correct. These are global non-constant variables, they should be snake case, as is they look like type names.

Reply:
happy to change, what's your recommendation for this?

Reply:
Snake case, so alias_lock, etc.